pkg/auth/requests/sar/sar.go

```diff
@@ -58,6 +60,14 @@ func (sar subjectAccessReview) UserCanImpersonateExtras(req *http.Request, user
 	return sar.checkUserCanImpersonateExtras(req.Context(), userContext, user, impExtras)
 }
 
+func (sar subjectAccessReview) UserCanImpersonateServiceAccount(req *http.Request, user string, sa string) (bool, error) {
```

Comment:
Smallest of nits:  document exported function

Reply:
done